states/huntsman/calibrating.py

```diff
@@ -82,9 +82,9 @@ def on_enter(event_data):
                 broad_band_cameras = list()
                 for cam_name, cam in pocs.observatory.cameras.items():
                     if cam.filter_type.lower().startswith('ha'):
```

Comment:
This is, of course, a horrible, hacky, unreliable way to identify narrowband filters but that's nothing to do with this PR. I think I wrote this...